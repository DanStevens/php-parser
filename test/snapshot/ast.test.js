const parser = require('../../src/index');

it('fix #127 - echo statements', () => {
  expect(parser.parseEval('echo "hello"; ?> world')).toMatchSnapshot();
});

it('fix #127 - inline', function() {
  expect(parser.parseEval('?>?>?>')).toMatchSnapshot();
});

it('test program', function() {
  expect(parser.parseEval('')).toMatchSnapshot();
});
