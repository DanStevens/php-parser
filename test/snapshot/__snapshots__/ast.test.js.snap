// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fix #127 - echo statements 1`] = `
Program {
  "children": Array [
    Echo {
      "arguments": Array [
        String {
          "isDoubleQuote": true,
          "kind": "string",
          "raw": "\\"hello\\"",
          "value": "hello",
        },
      ],
      "kind": "echo",
      "shortForm": false,
    },
    Inline {
      "kind": "inline",
      "raw": " world",
      "value": " world",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`fix #127 - inline 1`] = `
Program {
  "children": Array [
    Inline {
      "kind": "inline",
      "raw": "?>?>",
      "value": "?>?>",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`test program 1`] = `
Program {
  "children": Array [],
  "errors": Array [],
  "kind": "program",
}
`;
